steps/05/package.json

```diff
@@ -8,7 +8,7 @@
     "start": "ui5 serve -o index.html"
   },
   "devDependencies": {
-    "@types/openui5": "^1.120.0",
+    "@types/openui5": "^1.131.0",
```

Comment:
@matz3 @akudev There is a openui5 version `1.131.1` but no patch release for the types. So this means, there were no change in regards to types, right?

Reply:
@flovogt no, for @types/openui5, the release process is manual and quite some effort, so we do in general not release any patches. Only in rare exceptions.
For @openui5/types, on the other hand, the types are released ALWAYS as part of the release process, regardless of any changes.

Reply:
Thanks @akudev. I am in favour of migrating the project to `@openui5/types` . Then its also easier to switch to the `@sapui5/types`. Whats your opinion? @akudev @petermuessig